### PR TITLE
Django versatileimagefield

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/django-versatileimagefield/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
```
Comment (synapticarbors) [inline recipes/django-versatileimagefield/meta.yaml]: Is `pillow` a run requirement?

https://github.com/respondcreate/django-versatileimagefield/blob/master/setup.py#L18

File: recipes/django-versatileimagefield/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - versatileimagefield
```
Comment (synapticarbors) [inline recipes/django-versatileimagefield/meta.yaml]: Add:

```
   - versatileimagefield.datastructures
```

File: recipes/django-versatileimagefield/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+about:
+  home: https://github.com/respondcreate/django-versatileimagefield
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (synapticarbors) [inline recipes/django-versatileimagefield/meta.yaml]: Add issue/PR upstream for the license to be included and then add url to issue in the recipe as a comment.

Comment (sannykr) [reply recipes/django-versatileimagefield/meta.yaml]: @synapticarbors : Thanks. I have made all changes that you suggested. I included the license in the manifest.in file and made a pull request, and also added an issue about that.

**Comment by sannykr** [comment]
@synapticarbors : I don't know why it says that pillow is not in linux64 channel. I checked on conda-forge, and it is there. Could you please look at that.

**Pull request merged**